student.rb

```diff
@@ -1,16 +1,20 @@
-class Student
+require_relative "education"
+
+class Student < Education
   attr_reader :age, :phase
   attr_accessor :name
 
+  # PHASE = 1
+
   def initialize(options = {})
     @phase = 1
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    # @age = options.fetch(:age, 0)
+    # @name = options.fetch(:name, '')
   end
```

Comment:
You're overwriting the initialize in the superclass, so you aren't going to get the age and name!